e2e/images_test.go

```diff
@@ -42,6 +37,12 @@ func insertBundles(t *testing.T, cmd icmd.Cmd, dir *fs.Dir, info dindSwarmAndReg
 	return digest
 }
 
+func expectImageListOutput(t *testing.T, cmd icmd.Cmd, output string) {
```

Comment:
Maybe we can rename this one to `expectImageCommandOutput`? Since it's used in the tag tests too.

Reply:
The command specifically tests the output of `image list`, even if it's use in tag tests.